monai/losses/perceptual.py

```
@@ -205,11 +225,23 @@ class MedicalNetPerceptualSimilarity(nn.Module):
     """
 
     def __init__(
-        self, net: str = "medicalnet_resnet10_23datasets", verbose: bool = False, channel_wise: bool = False
+        self,
+        net: str = "medicalnet_resnet10_23datasets",
+        verbose: bool = False,
+        channel_wise: bool = False,
+        cache_dir: str | None = None,
     ) -> None:
         super().__init__()
         torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
```

Ruff (0.14.8)

Unused lambda argument: `a`

(ARG005)